### PR TITLE
fix checkstyle of core-task issue

### PR DESCRIPTION
**Pull request by seagle-yuan**
fix checkstyel issue

com.baidu.hugegraph.HugeFactory.java
com.baidu.hugegraph.HugeGraph.java
com.baidu.hugegraph.HugeGraphParams.java
com.baidu.hugegraph.StandardHugeGraph.java
com.baidu.hugegraph.io.HugeGraphSONModule.java
com.baidu.hugegraph.job.computer.AbstractComputer.java
com.baidu.hugegraph.job.computer.LouvainComputer.java
com.baidu.hugegraph.job.system.DeleteExpiredIndexJob.java 
com.baidu.hugegraph.plugin.HugeGraphGremlinPlugin.java
com.baidu.hugegraph.schema.builder.PropertyKeyBuilder.java
com.baidu.hugegraph.security.HugeSecurityManager.java
com.baidu.hugegraph.task.HugeTask.java
com.baidu.hugegraph.task.TaskManager.java
com.baidu.hugegraph.task.TaskScheduler.java

### CODE REVIEW

File: hugegraph-core/src/main/java/com/baidu/hugegraph/job/computer/AbstractComputer.java [hunk 1]
```diff
@@ -117,7 +114,7 @@ public Object call(Job<Object> job, Map<String, Object> parameters) {
             Process process = builder.start();
 
             StringBuilder output = new StringBuilder();
-            try(LineNumberReader reader = new LineNumberReader(
+            try (LineNumberReader reader = new LineNumberReader(
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/job/computer/AbstractComputer.java:117]: also update alignment?

File: hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java [hunk 2]
```diff
@@ -47,10 +47,10 @@ public class HugeSecurityManager extends SecurityManager {
             "org.apache.tinkerpop.gremlin.groovy.jsr223.GremlinGroovyScriptEngine"
     );
 
-    private static final Set<String> DENIED_PERMISSIONS = ImmutableSet.of(
-            "setSecurityManager"
-            //"suppressAccessChecks"
-    );
+    //"suppressAccessChecks"
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java:50]: expect to add a space after "//"

File: hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/PropertyKeyBuilder.java [hunk 3]
```diff
@@ -69,8 +68,7 @@ public PropertyKeyBuilder(SchemaTransaction transaction,
         this.checkExist = true;
     }
 
-    public PropertyKeyBuilder(SchemaTransaction transaction,
-                              HugeGraph graph, PropertyKey copy) {
+    public PropertyKeyBuilder(SchemaTransaction transaction, HugeGraph graph, PropertyKey copy) {
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/PropertyKeyBuilder.java:71]: ditto

File: hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/PropertyKeyBuilder.java [hunk 4]
```diff
@@ -43,8 +43,7 @@
 import com.baidu.hugegraph.type.define.WriteType;
 import com.baidu.hugegraph.util.E;
 
-public class PropertyKeyBuilder extends AbstractBuilder
-                                implements PropertyKey.Builder {
+public class PropertyKeyBuilder extends AbstractBuilder implements PropertyKey.Builder {
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/PropertyKeyBuilder.java:46]: can we keep 80 chars as guide line length?

File: hugegraph-core/src/main/java/com/baidu/hugegraph/job/computer/AbstractComputer.java (hunk 1 above)
Comment (seagle-yuan) [reply hugegraph-core/src/main/java/com/baidu/hugegraph/job/computer/AbstractComputer.java:117]: add whitespace after "try" word

Comment (javeme) [reply hugegraph-core/src/main/java/com/baidu/hugegraph/job/computer/AbstractComputer.java:117]: also update alignment of line 118

File: hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/PropertyKeyBuilder.java [hunk 5]
```diff
@@ -44,7 +44,7 @@
 import com.baidu.hugegraph.util.E;
 
 public class PropertyKeyBuilder extends AbstractBuilder
-                                implements PropertyKey.Builder {
+        implements PropertyKey.Builder {
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/PropertyKeyBuilder.java:47]: prefer to keep old style

File: hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/PropertyKeyBuilder.java [hunk 6]
```diff
@@ -69,8 +69,8 @@ public PropertyKeyBuilder(SchemaTransaction transaction,
         this.checkExist = true;
     }
 
-    public PropertyKeyBuilder(SchemaTransaction transaction,
-                              HugeGraph graph, PropertyKey copy) {
+    public PropertyKeyBuilder(SchemaTransaction transaction, HugeGraph graph,
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/PropertyKeyBuilder.java:72]: prefer to keep old style (don't exceed guide line length 80 chars)

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Pull request merged**